vendor/decentlab/index.yaml

```diff
@@ -44,3 +44,24 @@ endDevices:
   - dl-wrm
   - dl-zn1
   - dl-zn2
+profileIDs:
+  '1':
+    endDeviceID: dl-zn2
+    firmwareVersion: '1.5.71'
+    hardwareVersion: '1'
+    region: EU863-870
+  '2':
+    endDeviceID: dl-zn2
+    firmwareVersion: '1.5.71'
+    hardwareVersion: '1'
+    region: US902-928
+  '3':
+    endDeviceID: dl-zn2
+    firmwareVersion: '1.5.71'
+    hardwareVersion: '1'
+    region: AS923
```

Comment:
But this references exactly one device

In this case, there should simply be a profile and codec ID be defined. If there are multiple different codecs used, the number should not be defined in this file.

Reply:
Yeah it comes up often that its a single profile used by multiple devices but also multiple codecs. So how do i approach the numbering? you say `should not be defined in this file` but can you give me an example?